hermes-benchmark/src/jmh/java/pl/allegro/tech/hermes/benchmark/environment/InMemoryTopicsCache.java

```diff
@@ -0,0 +1,49 @@
+package pl.allegro.tech.hermes.benchmark.environment;
+
+import pl.allegro.tech.hermes.api.Topic;
+import pl.allegro.tech.hermes.common.kafka.KafkaTopic;
+import pl.allegro.tech.hermes.common.kafka.KafkaTopicName;
+import pl.allegro.tech.hermes.common.kafka.KafkaTopics;
+import pl.allegro.tech.hermes.common.metric.HermesMetrics;
+import pl.allegro.tech.hermes.frontend.cache.topic.TopicsCache;
+import pl.allegro.tech.hermes.frontend.metric.CachedTopic;
+
+import java.util.List;
+import java.util.Optional;
+
+import static pl.allegro.tech.hermes.api.ContentType.AVRO;
+import static pl.allegro.tech.hermes.test.helper.builder.TopicBuilder.topic;
+
+class InMemoryTopicsCache implements TopicsCache {
+
+    private final HermesMetrics hermesMetrics;
+    private final KafkaTopics kafkaTopics;
+    private final Topic topic = topic(HermesServerEnvironment.BENCHMARK_TOPIC).withContentType(AVRO).build();
```

Comment:
It would be clearer to explicitly define in one place what topic we use in the benchmarks. In `BenchmarkProviders` we create an instance of `InMemorySchemaClient` and say what topic it is going to use. Let's do the same with `InMemoryTopicsCache`.